radio_db/models.py

```diff
@@ -7,7 +7,9 @@ class BaseModel:
     """
     # TODO: implement base functionality that should fit for all models
 
-    def __init__(self, collection):
+    def __init__(self, db, collection):
+        # Should use one db param and name of collection
```

Comment:
Nice. In that way wed don't need to pass db every time. I was also thinking to pass app as param everywhere :D but that's another story